app/controllers/idv/phone_controller.rb

```diff
@@ -30,6 +30,11 @@ def new
     def create
       result = idv_form.submit(step_params)
       analytics.idv_phone_confirmation_form_submitted(**result.to_h)
+      irs_attempts_api_tracker.idv_phone_submitted(
```

Comment:
Since the phone number is evaluated asynchronously, are we interested in this initial submission (which does some simple phone validation) or the result of the asynchronous proofing, which occurs below in `async_state_done` ?

Reply:
May be a separate event I think. Wanting when a user enters in data, and then maybe a second one for the proofing result.

Reply:
I would expect the calls within `idv_form.submit(step_params)` to be blocking. Is there a point where they are not?

I will review the calls furher.

Reply:
For this particular event, the story is just to capture that a phone number has been submitted, so this looks correct to me. More of the proofing flow is handled in other tickets.